index.php

```diff
@@ -12,15 +13,14 @@ function tr($el) {
 
 
 function constructLink($type, $link) {
```

Comment:
This function is called 3x per entry on the website. That's a lot of entries in the cache. If you save the "optimized" HTML into the cache above, we only do this once every minute, which would help with how often we have to hit the cache with queries.

Reply:
Now the cache is only hit on parsing. Since the shortlinks don't expire we would expect lots of cache hits but way less than before :+1: